produce/lib/produce/merchant.rb

```diff
@@ -0,0 +1,83 @@
+require 'spaceship'
+
+module Produce
+  class Merchant
+    def create(_options, _args)
+      login
+
+      merchant_identifier = Produce.config[:merchant_identifier] || UI.input("Merchant identifier (reverse-domain name style string starting with 'merchant'): ")
+
+      if merchant_exists? merchant_identifier
+        UI.success("[DevCenter] Merchant '#{merchant_identifier})' already exists, nothing to do on the Dev Center")
+      else
+        merchant_name = Produce.config[:merchant_name] || merchant_identifier.split(".").map(&:capitalize).reverse.join(' ')
```

Comment:
@ohayon This was already used in https://github.com/fastlane/fastlane/blob/master/produce/lib/produce/group.rb#L17 as a way of creating a sensible default name from the provided identifier so I copied it here. 

ie. merchant.com.example.app.sandbox is given a name of "Sandbox App Example Com Merchant" if no specific merchant_name was provided

 I'm happy to change it or add a comment in code to give more detail if you like?